src/univariate/continuous/generalizedinversegaussian.jl

```diff
@@ -0,0 +1,273 @@
+doc"""
+    GeneralizedInverseGaussian(a,b,p)
+
+The *generalized inverse Gaussian distribution* with parameters a > 0, b > 0, p real,
+and modified Bessel function of the second kind K_p, has probability density function
+
+$f(x; a, b, p) = \frac{(a/b)^{p/2}}{2K_p(\sqrt{ab})}x^{(p-1)}
```

Comment:
Math should be surrounded with double backticks rather than the TeX-style dollar signs